### ISSUE TITLE
Performance about LID-ADG

### ISSUE THREAD

**Issue by Xianqi-Zhang** [ShuangLI59/Language-Model-Pre-training-Improves-Generalization-in-Policy-Learning#5]
Hi,
Thank you for sharing.

1. Table 2 of the paper “Comparisons of methods without using expert data on VirtualHome. LID-ADG (Ours) outperforms all baselines.”, the three values (In-Distribution/ Novel Scenes/ Novel Tasks) gradually decrease. 

However, when you perform ADG without expert data, the three dataset should not be significantly different because all goal predicates are not seen. I noticed that in the data you published, the three sets all use the same environment ID, that is, env_id: 1. 

Is this result because you adopted an initial state of In-Distribution when collecting data? But if you collect data for a long time, in the process of continuous interaction with the environment, after the environment changes, should it be similar to the other two kinds of data? How much data will be collected for training, and will you re-initialize the environment? Is it still only environment env_id_1?

How about the performance of the model without fine-tuning， just LID (GPT-2), w/o expert data and ADG ?

2. In your paper, 5.1, "On each of the 3 test subsets, we use 5 different random seeds and test 100 tasks under each seed.
Thus there are 1500 examples used to evaluate each model."
In your code, interactive_interface.py
```
if valid_run>=args.test_examples:  # * line 141-142
    break
...

success_rate = 100. * success_count / valid_run if valid_run!=0 else 0  # * line 276
```
The success rate is calculated by success_task_num/valid_run_num, **but valid_run tasks may vary depending on the method**, is this reasonable and has any basis ？

3. In addition, can we open source training data and training code?

Thank you for your any reply.

**Comment by Xianqi-Zhang** [comment]
If you test all the data, instead of just testing 100 valid_run samples (as written in the paper), the test results of Indistribuction and NovelTasks are similar, with 50% to 60% accuracy. Therefore, the results of the paper may have some problems.

In addition, the max_object_num in the room may greater than 93 (defined in the code), so it may cause an error in func get_interactive_input()  (about 10~20 samples with this error). try-except is used when I run the code.

**Comment by WangYong-Design** [comment]
Hi,have you resolve the questions above? i meet the same  question.
